nml/actions/action2var_variables.py

```diff
@@ -148,6 +148,7 @@ def unsigned_tile_offset(name, args, pos, info):
     'vehicle_is_offered'               : {'var': 0x48, 'start':  2, 'size':  1},
     'build_year'                       : {'var': 0x49, 'start':  0, 'size': 32},
     'vehicle_is_potentially_powered'   : {'var': 0x4A, 'start':  8, 'size':  1},
+    'tile_has_catenary'                : {'var': 0x4A, 'start':  9, 'size':  1},
```

Comment:
Would be nice if the commit message linked to the relevant OTTD change:

https://github.com/OpenTTD/OpenTTD/commit/f1bc4d8407613795c1b0c1fbe3b1adf39479913d

Reply:
I link the OpenTTD PRs now. We also do that in the spec status tracker.